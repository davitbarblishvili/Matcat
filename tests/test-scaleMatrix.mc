// Author: Davit
func main() int {
    matrix a; 
    matrix b; 

    a = [[1,2,3],[4,5,6]];
    b = 3 * a;

    printm(b);

    return 0;
}