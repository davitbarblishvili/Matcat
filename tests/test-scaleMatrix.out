(
[3.00 6.00 9.00 ]
[12.00 15.00 18.00 ]
)